schema.json

```diff
@@ -316,59 +301,158 @@
           "type": "string"
         },
         "lastModified": {
-          "description": "e.g. 2017-06-29T15:53:00 - [resume.json uses the ISO 8601 date standard]",
+          "description": "date-time of last modified",
           "format": "date-time",
           "type": "string"
         },
+        "uuid": {
+          "description": "uuid v4 of the resume",
+          "type": "string"
+        },
         "version": {
-          "description": "e.g. v1.0.0 - [A version field which follows semver]",
+          "description": "version field which follows semver",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "personal": {
+      "description": "sensitive informations",
+      "properties": {
+        "birthPlace": {
+          "description": "place of birth",
+          "type": "string"
+        },
+        "birthday": {
+          "description": "birthday date",
+          "format": "date",
+          "type": "string"
+        },
+        "currentLocation": {
+          "description": "living location",
+          "properties": {
+            "lat": {
+              "type": "number"
+            },
+            "long": {
+              "type": "number"
+            }
+          },
+          "type": "object"
+        },
+        "email": {
+          "description": "email address",
+          "format": "idn-email",
+          "type": "string"
+        },
+        "gender": {
+          "type": "string"
+        },
+        "image": {
+          "description": "url of the personal photo",
+          "type": "string"
+        },
+        "name": {
+          "description": "full name",
+          "type": "string"
+        },
+        "permanentLocation": {
+          "description": "permanently living location",
+          "properties": {
+            "lat": {
+              "type": "number"
+            },
+            "long": {
+              "type": "number"
+            }
+          },
+          "type": "object"
+        },
+        "phone": {
+          "description": "phone number",
+          "type": "string"
+        },
+        "postalAddress": {
+          "type": "string"
+        },
+        "profiles": {
+          "description": "list of social networks",
+          "items": {
+            "properties": {
+              "network": {
+                "description": "name of the network",
+                "type": "string"
+              },
+              "url": {
+                "description": "url of the profile",
+                "format": "uri",
+                "type": "string"
+              },
+              "username": {
+                "description": "username in the network",
+                "type": "string"
+              }
+            },
+            "type": "object"
+          },
+          "type": "array"
+        },
+        "relationshipStatus": {
+          "description": "civil status",
+          "type": "string"
+        },
+        "summary": {
```

Comment:
I think that summary might not be identifiable, but I might be missing something. What was the specific reasoning behind adding this into personal?